server/game/cards/events/07/therearenomenlikeme.js

```diff
@@ -0,0 +1,32 @@
+const DrawCard = require('../../../drawcard.js');
+
+class ThereAreNoMenLikeMe extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: "Select character that doesn't kneel for military challenges",
+            target: {
+                activePromptTitle: 'Select a Knight character',
+                cardCondition: card => card.location === 'play area' && card.getType() === 'character' && card.hasTrait('knight')
+            },
+            handler: context => {
+                this.untilEndOfPhase(ability => ({
+                    match: context.target,
+                    condition: () => (
+                        this.game.currentChallenge &&
+                        this.game.currentChallenge.challengeType === 'military'
+                    ),
+                    effect: [
+                        ability.effects.doesNotKneelAsAttacker(),
+                        ability.effects.doesNotKneelAsDefender()
+                    ]
+                }));
+
+                this.game.addMessage('{0} uses {1} to keep {2} standing when participating in military challenges this phase.', context.player, this, context.target);
```

Comment:
This is super minor, but the character can still be knelt by card effects when participating during military challenges. Maybe something like "{0} uses {1} to have {2} not kneel when declared as an attacker or defender in military challenges until the end of the phase".

Also, commits from previous PR's seem to be included here.